Script/version.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+set -e
+
+NC='\033[0m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+
+echo "Target version - ${BLUE}$1${NC}"
+echo "------------------AEPRoku-------------------"
+SOURCE_CODE_VERSION=$(cat ./code/main/common/version.brs | egrep '\s*VERSION\s*=\s*\"(.*)\"' | sed -e 's/.*= //; s/,//g; s/"//g')
```

Comment:
Let's verify the version number inside the final merged SDK file.

Reply:
This is the release script, not the PR verification script. I would rather verify the merge brs file instead of the source file.